test/Serilog.Tests/LoggerConfigurationTests.cs

```diff
@@ -345,5 +345,23 @@ public void LevelOverridesArePropagated()
 
             Assert.Equal(2, sink.Events.Count);
         }
+
+        [Fact]
+        public void LevelOverridesDownArePropagated()
```

Comment:
Naming? `LevelOverridesDownArePropagated` to `LowerMinimumLevelOverridesArePropagated` & `LevelOverridesUpArePropagated ` to `HigherMinimumLevelOverridesArePropagated`